invenio_files_rest/ext.py

```diff
@@ -45,21 +45,6 @@ def __init__(self, app):
         self.app = app
 
     @cached_property
-    def record_file_factory(self):
-        """Load default storage factory."""
-        r = load_or_import_from_config(
-            'FILES_REST_RECORD_FILE_FACTORY', app=self.app)
```

Comment:
``FILES_REST_RECORD_FILE_FACTORY`` should be removed from ``config.py`` as well + perhaps a comment in the commit that it's removed.